source/includes/_intro.md

```diff
@@ -238,6 +238,77 @@ like `user name` or `account_number`, without specifying that you are searching
 *Pagination*      | x | x | x | x | x | x | x | x
 *Sorting*         | x | x | x | x | x | x | x | x
 
+
+### Search model by tags
+
+Some `GET` methods allowing you to search models by *tags*.
+Above you will see how could you use this features.
+
+There are 3 ways of getting models by tags:
+
+- Get all models contain the tag(s) in their list of tags.
+
+*Code: `?tags=tag_1`*
+
+- Get all models contain the combination of multiple tags in their list of tags.
+
+*Code: `?tags=tag_1+tag_2`*
+
+- Get all models contain a specific tag in their list of tags.
+
+*Code: `?tags=tag_1+`*
+
+Let's named a value in the `tags` parameter is an expression like  `tag_1` in `?tags=tag_1`.
+
+Basically the structure if `some-route?tags=some-tag` each expression has to be separated by a `,` and an expression of a combination of tags needs a `+` like `tag_1+tag_2`.
+You can fill `tags` with multiple and different expressions, see on the following examples:
```

Comment:
I suggest:
```
Under follows some examples on how to query the models by tags.

The basic structure when querying a model by tags is `some-route?tags=some-tag`. 

It is also possible to query by multiple tags, like `some-route?tags=tag_1,tag_2`. This will return every document that matches either `tag1_` or `tag_2`, or both.

If you only want models that match both `tag1_` and `tag_2`, separate the tags by `+`. Like `some-route?tags=tag_1+tag_2`.

```